packages/frontend-2/lib/viewer/composables/ui.ts

```diff
@@ -643,54 +646,31 @@ export function useMeasurementUtilities() {
     state.ui.measurement.options.value = options
   }
 
-  const removeMeasurement = () => {
```

Comment:
same problem viewModes had - state creation, the actual integration and helpers/utilities are all mixed together and needed refactoring so that each of those happens in its own dedicated computed